src/command.c

```diff
@@ -204,10 +206,30 @@ handle_command_stop (const struct subcommand *sub,
 	}
 
 	ret = cc_oci_stop (config, state);
+
+        if (state->devices) {
+                /* loading appropriate drivers on the devices */
+		g_debug ("loading drivers on the devices that pass-thru'ed");
```

Comment:
Hahaha -- Yes, will do this, agreed - some of the wording bothered me before, but I wanted someone from England to point out the English :)

You have any favorite (or is it favourite) spell checker for source?